Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -336,70 +331,81 @@ - (DisplayableText *)displayableTextForInteractionId:(NSString *)interactionId
     return displayableText;
 }
 
+- (nullable TSAttachment *)firstAttachmentIfAnyOfMessage:(TSMessage *)message
+{
+    if (message.attachmentIds.count == 0) {
+        return nil;
+    }
+    NSString *_Nullable attachmentId = message.attachmentIds.firstObject;
+    if (attachmentId.length == 0) {
+        return nil;
+    }
+    return [TSAttachment fetchObjectWithUniqueID:attachmentId];
+}
+
 - (void)ensureViewState
 {
-    OWSAssert([self.interaction isKindOfClass:[TSMessage class]]);
+    OWSAssert([self.interaction isKindOfClass:[TSOutgoingMessage class]] ||
+        [self.interaction isKindOfClass:[TSIncomingMessage class]]);
```

Comment:
Is this significant? 😱

Reply:
Not really, I just realized that we might someday accidentally use an `OWSMessageCell` for an info or error message.  This assert will quickly expose that class of bug.

Reply:
👍